src/Sylius/Bundle/WebBundle/Resources/views/Backend/Inventory/macros.html.twig

```diff
@@ -72,7 +72,7 @@
                                     <td>
                                         <ul>
                                         {% for option in variant.options %}
-                                            <li><strong>{{ option.name }}</strong>: {{ option.value }}</li>
+                                            <li><strong>{{ option.presentation }}</strong>: {{ option.value }}</li>
```

Comment:
It wasn't. It was throwing exception. Twig naming is misleading - here the option is `OptionValue` which now doesn't have a `name` only `value`. Presentation will get the name from assocatiated `Option` see codeblock in description.